sub/source/docs/pilot-control.rst

```diff
@@ -0,0 +1,55 @@
+.. _pilot-control:
+
+=============
+Pilot Control
+=============
+
+Control of the vehicle can be accomplished in three ways:
+
+- Via Ground Control Stations using:
+   - GCS attached Joysticks, or command lines, to send MAVLink `RC override <https://mavlink.io/en/messages/common.html#RC_CHANNELS_OVERRIDE>`__ commands that act as RC inputs (even if no RC receiver is attached)
+   - MAVLink ` MANUAL CONTROL <https://mavlink.io/en/messages/common.html#MANUAL_CONTROL>`__ messages which includes axis commands and button states.
+
+- RC control (not enabled by default)
+
+Pilot Control Inputs
+====================
+If RC is not enabled, pilot inputs are via the GCS joystick, controlling the vehicle axes, and body-frame movements laterally,forward/back, and up/down in depth. Additionally, lights and camera can be controlled from the GCS.
+
+If RC is enabled, then the pitch/roll/yaw/throttle/lateral/forward inputs are sent from the RC transmitter and mapped from RC channels to these inputs via the ``RCMAP_`` parameters.
+
+Pilot Control Freedoms by Mode
+==============================
+The following assumes the axis is controllable. Some frame configurations do not provide all axes.
+
+ANG = Angle target. Stick deflection indicates desired axis angle
+
+RAT = Rate target. Sitck deflection indicated desired axis movement/rotation rate.
+
+D = Direct motor output based on stick deflection
+
+=========== ===== ===== ==== ===== ======= ======= ======= =========
+Mode        Roll  Pitch Yaw  Depth Forward Lateral PosHold DepthHold
+=========== ===== ===== ==== ===== ======= ======= ======= =========
+MANUAL       D     D     D     D     D       D       No      No
+ACRO         RAT   RAT   RAT   RAT   RAT     RAT     No      No
+STABILIZE    ANG   ANG   RAT   RAT   RAT     RAT     No      No
+ALT_HOLD     ANG   ANG   RAT   RA    RAT     RAT     No      Yes
+POSHOLD      ANG   ANG   RAT   RAT   RAT     RAT     Yes     Yes
```

Comment:
`POSHOLD` should potentially be moved one lower, as it introduces additional automation capabilities beyond what `SURFTRAK` provides.

Reply:
done